src/nerdbank-streams/asyncprocess.ts

```diff
@@ -0,0 +1,18 @@
+import * as cp from "child_process";
+import * as Promise from "promise";
+
+export interface IExecAsyncResult {
```

Comment:
I can see how that's true. C++ header files are at least similar in they tend to aggregate the public API in one place. C# style is to apply the visibility to the type/member itself, which I prefer since I can at a glance see if it's safe to change this member without having to scroll to the bottom of the file or to the header file. So for now I'll keep it this way, but happy to chat about it more some time.